python/Dockerfile

```diff
@@ -1,8 +1,16 @@
 FROM alpine
 
+RUN apk add --no-cache python3 py3-pip
```

Comment:
There are some things you can do to remove temporary files from the image.
Some are described here specifically if you use apk.
https://octopus.com/blog/using-alpine-docker-image